cleanup-lambda/src/handler.ts

```diff
@@ -0,0 +1,21 @@
+import { TABLE_NAME } from "./config";
+import { createDbConnection } from './rds';
+
+export const main = async (): Promise<void> => {
+	const sql = await createDbConnection();
+
+	try {
+		const result = await sql`
+            DELETE
+            FROM ${sql(TABLE_NAME)}
+            WHERE ingested_at < NOW() - INTERVAL '14 days'
+                RETURNING *;
+        `;
+
+		console.log(`Deleted ${result.count} records`);
+	} catch (error) {
+		console.error('Error deleting old records:', error);
```

Comment:
might we want to fail the job in this case, so that we can add retry behaviour? (this could wait for a separate PR though)